scenes/moving_platform.tscn

```diff
@@ -0,0 +1,18 @@
+[gd_scene load_steps=2 format=3 uid="uid://bl5nxncigcwgt"]
+
+[sub_resource type="Curve2D" id="Curve2D_eqwmb"]
+_data = {
+"points": PackedVector2Array(0, 0, 0, 0, 1155, 3, 0, 0, 0, 0, 0, 0)
+}
+point_count = 2
+
+[node name="MovingPlatform" type="Node2D"]
+
+[node name="Path2D" type="Path2D" parent="."]
```

Comment:
where is this?

Reply:
this is not used, probably no effect

![image](https://github.com/Dragonlinae/Papereaven/assets/12877445/6f57c1ce-679f-47e2-8060-62ceb1fb16b2)